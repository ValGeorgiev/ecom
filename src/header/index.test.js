import React from 'react'
import Header from './index'
import renderer from 'react-test-renderer'

jest.mock("react-router-dom", () => ({
  Link: "Link"
}));

describe('Header component', () => {
  test('should render correctly', () => {
    const component = renderer.create(
      <Header />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})