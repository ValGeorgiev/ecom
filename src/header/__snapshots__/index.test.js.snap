// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render correctly 1`] = `
<div
  className="header-container"
>
  <Link
    className="home-link"
    to="/"
  >
    SoftUni React.js course
  </Link>
  <Link
    className="checkout-link"
    to="/checkout"
  >
    Checkout
  </Link>
  <input
    onChange={[Function]}
    placeholder="Search..."
    value=""
  />
</div>
`;
